frostsnapp/README.md

```diff
@@ -0,0 +1,34 @@
+# frostsnapp
+
+This repository serves as a template for Flutter projects calling into native Rust
+libraries via `flutter_rust_bridge`.
+
+## Install
+
+To begin, ensure that you have a working installation of the following items:
+- [Flutter SDK](https://docs.flutter.dev/get-started/install)
+- [Rust language](https://rustup.rs/)
+- `flutter_rust_bridge_codegen` [cargo package](https://cjycode.com/flutter_rust_bridge/integrate/deps.html#build-time-dependencies) -- **Make sure to install the same version as in the project's [`Cargo.lock`](../Cargo.lock)**  
```

Comment:
Note: on non-debian linux you need to set `export CPATH="$(clang -v 2>&1 | grep "Selected GCC installation" | rev | cut -d' ' -f1 | rev)/include"`